app/controllers/hub/documents_controller.rb

```diff
@@ -10,6 +10,8 @@ class DocumentsController < ApplicationController
     layout "admin"
 
     def index
+      ConsentPdf.new(@client.intake).create_consent_document if Document.where(document_type: "Consent Form 14446", intake: @client.intake).count.zero?
```

Comment:
This is a database write on a HTTP GET. Can we move the DB write somewhere else perhaps?

Reply:
I'd prefer that we run a one-time script to generate these for clients that have not had them generated yet this season rather than generating them when hitting the documents index view.